specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/ManagedInstances.json

```diff
@@ -88,7 +88,7 @@
             }
           },
           "default": {
```

Comment:
>"default": { [](start = 10, length = 12)

The "default" response should have a associated schema.

Reply:
@filizt 
do we have ARM/API/SDK rules to enforce it? All these spec and example files are auto-generated by SQL DB repo. And this issue was not mentioned in any previous PRs.

Reply:
Hi @ericshape , actually we have ARM Lint rule for this: https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/openapi-authoring-automated-guidelines.md#r4007-defaulterrorresponseschema

But currently to keep your service consistent and breaking change consideration, this Rule is only for new Service, guess why Filizt said ideally. :)

Reply:
Thanks for your comments, Arthur! 
It seems the static validation service is mandatory for SQL DB repo. It will enforce these rules on Day one and reduce the engineer efforts on both ARM/API/SDK and SQL DB sides.